AstSemantics.md

```diff
@@ -250,9 +250,16 @@ Branches that exit a `block`, `loop`, or `tableswitch` may take a subexpression
 that yields a value for the exited construct.
 Branches may only reference labels defined by an outer *enclosing construct*.
 This means that, for example, references to a `block`'s label can only occur 
-within the `block`'s body. In practice, outer `block`s can be used to place labels for any
-given branching pattern, except for one restriction: one can't branch into the middle of a
-loop from outside it. This restriction ensures all control flow graphs are well-structured.
+within the `block`'s body.
+
+In practice, outer `block`s can be used to place labels for
+[any given branching pattern](Rational.md#control-flow),
```

Comment:
the only change here is a link. the rest is due to this pull being based off of #475 which has not landed yet.